vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -17,7 +17,7 @@ NODE_DRAIN_TIMEOUT: "{{ lookup('env', 'NODE_DRAIN_TIMEOUT') | default('0s', true
 BMOPATH: "{{ lookup('env', 'BMOPATH') }}"
 IRONIC_DATA_DIR: "{{ lookup('env', 'IRONIC_DATA_DIR') }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
-KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.21.0', true) }}"
+KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.21.1', true) }}"
 UPGRADED_K8S_VERSION: "{{ lookup('env', 'UPGRADED_K8S_VERSION') | default('v1.21.1', true) }}"
```

Comment:
This can be removed now?

Reply:
you mean the `UPGRADED_K8S_VERSION` ?

Reply:
Yeah, since we are passing it in dedicated *_vars.sh, I do not think we need it? It does not hurt functionality wise though.

Reply:
Actually, we consuming the variable here. I can remove the default value from here, though.

Reply:
removed the default value. 
it is added here in order to avoid repeating `lookup('env', 'UPGRADED_K8S_VERSION')` in the ansible tasks.